build.gradle

```diff
@@ -41,8 +41,13 @@ dependencies {
 }
 
 javafx {
-    version = "11.0.1"
-    modules = [ 'javafx.controls' ]
+    version = "11.0.2"
+    modules = [ 'javafx.controls',
+                'javafx.base',
```

Comment:
In a proper app I think you would only include the modules that are actually required. Looking at the app web, media and probably fxml are not required. Maybe a better approach would be to explain in the doc that you might need to add more modules depending on the functionality you use and also list all the modules that we know are used if you use every part of the API.

I think you can run jdeps or something on the JAR (might also work on a class) to get the full list. I can't remember the error message you get if you are missing a module but I think there is enough information in it to work out how to resolve it.

You can't win though - if you include them all someone will point out they aren't all needed and if you don't others will not know what to do when they add more API use and it doesn't compile.